pysand/fluidproperties.py

```diff
@@ -1,7 +1,23 @@
 import numpy as np
+import logging
+import pysand.exceptions as exc
 
+logger = logging.getLogger(__name__)
 # Models from DNVGL RP-O501, equation references in parenthesis
 
+def validate_fluid_props(**kwargs):
+    """
+        Validation of all input parameters that go into erosion models;
+    """
+    for i in ['P', 'T', 'Qo', 'Qw', 'Qg', 'Z', 'D', 'rho_o', 'rho_w', 'MW', 'mu_o', 'mu_w', 'mu_g']:
+        if i in kwargs:
+            if kwargs[i] is None:
+                raise exc.FunctionInputFail('No fluid properties are calculated due to missing {}'.format(i))
+            if not isinstance(kwargs[i], (float, int)):
```

Comment:
Do we not care about it being a `pd.series`?

Reply:
That's the thing we need to clarify. I thought as of now all these functions will be implemented via "apply" function. If we want them to have pd.Series or np.array as inputs, then I guess it needs some thorough changes. We should discuss it.